Client/Frontend/Home/FirefoxHomeTopSitesCell.swift

```diff
@@ -79,7 +79,7 @@ class TopSiteItemCell: UICollectionViewCell, Themeable {
         accessibilityIdentifier = "TopSite"
         contentView.addSubview(titleLabel)
         contentView.addSubview(faviconBG)
-        contentView.addSubview(imageView)
+        faviconBG.addSubview(imageView)
```

Comment:
curious why this layout needed to change?

Reply:
https://github.com/SnapKit/SnapKit/pull/569 changed how insets and centering is applied. This change just re-centers the favicon on the background.